### ISSUE TITLE
Metis Launcher - Rearming itself

### ISSUE THREAD

**Issue by TophatSpearhead** [iamthesux/KingsHorses#69]
Date: (4/1/15)

(x2) Metis Launchers were assembled at the NW Pillbox by COP Smirnoff. They were used to engage and destroy (x2) T-72 unit. Following this, they were used for spotting purposes for an MG team. At this time, the individual manning one discharged a rocket, believing the unit to be empty. Several seconds later, another was discharged. This was tested further using the second launcher.

In short, they are rearming themselves after a few seconds.

To reproduce, assemble a metis. Fire it, then wait a few seconds and attempt again.

**Comment by iamthesux** [comment]
That's odd,  I'll investigate.
